client/app/assets/less/redash/redash-newstyle.less

```diff
@@ -480,6 +490,86 @@ body {
   background-color: fade(@redash-gray, 12%) !important;
 }
 
+#topRefreshIndicator {
```

Comment:
Kebab-case please :pray: Also, don't use `id` for elements that may occur multiple times on page - they're not intended for this. Also, ids are very hard to override in CSS because they have highest priority and support the only value in attribute.

Reply:
40ec0c0